certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1661 @@
+      "email": "support@zohosign.com",
+      "url": "https://www.zoho.com/sign/contact.html"
+    }
+  },
+  "host": "sign.zoho.com",
+  "basePath": "/api/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/x-www-form-urlencoded"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/requests/{request_id}/completioncertificate": {
+      "get": {
+        "produces": [
+          "application/pdf"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "HTTP/1.1": {
+                "description": "",
+                "type": "string"
+              },
+              "Content-Type:application/pdf;": {
+                "description": "Content-Type:application/pdf;",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Download Completion Certificate",
+        "description": "Download the completion certificate",
+        "operationId": "download-completion-certificate",
+        "parameters": [
+          {
+            "name": "request_id",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "format": "int64",
+            "description": "Request ID",
+            "x-ms-summary": "Request ID",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/requests/{request_id}/pdf": {
+      "get": {
+        "produces": [
+          "application/pdf"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "HTTP/1.1": {
+                "description": "",
+                "type": "string"
+              },
+              "Content-Type:application/pdf;": {
+                "description": "Content-Type:application/pdf;",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Download Document",
+        "description": "Download the selected document",
+        "operationId": "download-document",
+        "parameters": [
+          {
+            "name": "request_id",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "format": "int64",
+            "x-ms-summary": "Request ID",
+            "description": "Request ID",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/requests/{request_id}/documents/{document_id}/pdf": {
+      "get": {
+        "produces": [
+          "application/pdf"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "HTTP/1.1": {
+                "description": "",
```

Comment:
@ZohoSign-integ Empty description values are not allowed. Please let us know the reason for adding this headers section with "HTTP/1.1".